astropy/units/tests/test_format.py

```diff
@@ -1051,7 +1051,7 @@ class TestUnknownFormat:
     # Check full message to ensure we correctly classify in-out and
     # output only formats.
     UNKNOWN_MSG = (
-        r"Unknown format {!r}.  Valid formatter names are: "
+        r"Unknown format {!r}. Valid formatter names are: "
```

Comment:
I removed a space from the error messages because otherwise the `raise ValueError(...)` line in the function would have been 89 characters long, which would have forced Ruff to split it over three lines.